src/components/calcite-action-bar/usage/basic.md

```diff
@@ -0,0 +1,53 @@
+#### Basic Action Bar (Top Actions)
+
+Renders `calcite-actions` just on the top half of the bar.
+
+```html
+<calcite-action-bar>
+  <calcite-action text="Add">
+    <!-- icon -->
+  </calcite-action>
+
+  <calcite-action text="Save">
+    <!-- icon -->
+  </calcite-action>
+</calcite-action-bar>
+```
+
+#### With grouping
+
+Renders a group of `calcite-actions` contained in a `calcite-action-group`.
+
+```html
+<calcite-action-bar>
+  <calcite-action-group>
+    <calcite-action text="Add">
+      <!-- icon -->
+    </calcite-action>
+    <calcite-action text="Save">
+      <!-- icon -->
+    </calcite-action>
+    <calcite-action text="Layers">
+      <!-- icon -->
+    </calcite-action>
+  </calcite-action-group>
+</calcite-action-bar>
+```
+
+#### Bottom Actions
+
+Renders `calcite-actions` that stick to the bottom of the action bar.
+
+```html
+<calcite-action-bar>
+  <div slot="bottom actions">
```

Comment:
Maybe have an action at the top too? And just one at the bottom.

Maybe also explain that they will be above the expand/collapse?